Core/HLE/sceNetAdhocMatching.cpp

```diff
@@ -630,14 +594,12 @@ void actOnHelloPacket(SceNetAdhocMatchingContext * context, SceNetEtherAddr * se
 				SceNetAdhocMatchingMemberInternal * peer = findPeer(context, sendermac);
 
 				// Peer not found
-				if (peer == NULL)
-				{
+				if (peer == NULL) {
```

Comment:
In the future, when changing bracing and finding `if (x == NULL)`, go ahead and change it to `if (!x)`.  

(only do this when changing the line anyway, don't go searching for `== NULL`)